### DISCUSSION TITLE
What is going on?

### DISCUSSION THREAD

**Discussion by kxnkxv** [jaredpalmer/tsdx#1150] [Q&A]
Hello everybody! Please tell us what is going on with this repository? Why have there been no releases for 2 years, even taking into account the presence of community support, and how to start the library using the "new" version?

**Comment by tk-o** [comment]
For anyone wondering on what is a good alternative for TSDX, have a look at:
- https://github.com/egoist/tsup

I think the fact that [the Turborepo docs mention `tsup`](https://turbo.build/repo/docs/handbook/publishing-packages/bundling) speaks for itself. Also, Turborepo was created by @jaredpalmer so I just connected the dots 🙃

**Comment by moishinetzer** [comment]
For anyone else who wants an alternative for TSDX but specifically geared to creating a **component library**, check out:

- https://github.com/moishinetzer/pbandj

It makes creating and publishing a high-quality component library really easy, and of course, zero-config.

(Its also fine for regular ts libs but it will have extra features geared specifically at component libraries)

**Reply by leamandeep** [reply]
Does this mean this library is no long to maintained? so is it safe to use?
